### PR TITLE
enhanced permissions fix command

### PR DESCRIPTION
**Pull request by jfmercer**
The previous "permissions fixer" command was 
```bash
sudo chown -R $(whoami) /usr/local
```
However, this is incomplete in two respects. First, it doesn't include the group (i.e., ```chown user:group```). Secondly, it assumes that the ```HOMEBREW_PREFIX``` is always ```/usr/local```, whereas, in rare cases, users choose a custom ```HOMEBREW_PREFIX```, such as ```/opt```, ```/opt/brew``` or whatever. 

Now, I suggest 
```bash
sudo chown -R $(whoami):admin $(brew --prefix)
```
to fix both problems. According to [the homebrew install script](https://raw.githubusercontent.com/Homebrew/install/master/install), the group should be ```admin```. This alias sets it to ```$USER:admin```. Secondly, just in case the user has a custom ```HOMEBREW_PREFIX```, this alias will fix permissions in the correct homebrew directory in all cases. 

If you have questions, please let me know. :-)

### CODE REVIEW

**Comment by mikemcquaid** [comment]
While your version is more correct it's also more complex and harder to understand. I think it's fine as-is, personally; I've not seen confusion result from the current version.

**Comment by jfmercer** [comment]
Point well taken. Would the following change be acceptable? If so, I'll update the PR. 
```bash
sudo chown -R $(whoami):admin /usr/local
```
Also, if you would like me to write additional documentation about this command, I would be happy to do so. Just let me know.

I thought of this PR based on my personal experience. For years, I would use ```sudo chown -R $(whoami):staff /usr/local```, or, far worse, ```sudo chown -R $(whoami):wheel /usr/local```. ```staff``` breaks at times. ```wheel``` works, of course, but it is reserved exclusively for ```root``` and should never, ever be used by a user, even an admin user. 

In short, the real point of this PR is to add the correct ```group``` to the ```chown``` command. 

Let me know if the suggested change would be acceptable and if you'd like me to write additional documentation explaining the command. 

If you really thing the PR is unneeded, just close it. No hard feelings. 

Thank you for your help.

**Comment by mikemcquaid** [comment]
I'll let other @Homebrew/owners weigh in but personally I don't see the need to change the group unless it's causing other problems. Will leave this open for now. Thanks and sorry to disagree!

**Comment by DomT4** [comment]
Hasn't been a huge amount of interest in this, so am going to close - Will review going forwards if it proves necessary.

Thanks for the submission, We appreciate it!
